rootfs/Dockerfile

```diff
@@ -13,13 +14,14 @@ RUN apt-get update \
 		python-dev \
 		--no-install-recommends \
 	&& curl -sSL https://bootstrap.pypa.io/get-pip.py | python - pip==8.1.1 \
-	&& pip install --disable-pip-version-check --no-cache-dir docker-py==1.8.1 \
+	&& pip install --disable-pip-version-check --no-cache-dir git+https://github.com/docker/docker-py.git@9b63bed6a0b5185b043e85df8c49d86d2c048aa1 \
```

Comment:
My patch is not in 1.9.0...it will be present in 1.10.0.
https://github.com/docker/docker-py/pull/1133#issuecomment-235662853